resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/StringParameterInjector.java

```diff
@@ -65,7 +65,7 @@ public StringParameterInjector(Class type, Type genericType, String paramName, C
       initialize(type, genericType, paramName, paramType, defaultValue, target, annotations, factory);
    }
 
-   public boolean isCollectionOrArray()
+   public boolean isInjectableAsCollectionOrArray()
    {

```

Comment:
Not a good idea to change the name of a public method. Even if it is, in principle, an implementation method, it is still visible outside of Resteasy.